(5a) More Luxuries INT (v 1)/XML/de_DE/NewLuxuries_Text.xml

```diff
@@ -5,35 +5,43 @@
 	<language_de_DE>
 		<!--
 		<Replace Tag="TXT_KEY_RESOURCE_COCOA">
-			<Text>Cocoa</Text>
+			<Text>Kakaio</Text>
```

Comment:
Needs to be Kakao. Unfortunately I am not allowed to edit the file directly, because I have no push access to your master branch.